packages/stable/src/api/spatial/types.ts

```diff
@@ -0,0 +1,141 @@
+import { CogniteExternalId, CogniteInternalId, Limit } from '@cognite/sdk-core';
+
+export type GeometryType = 'wkt' | 'geojson';
+
+export type AttributeType =
+  | 'STRING'
+  | 'LONG'
+  | 'DOUBLE'
+  | 'BOOLEAN'
+  | 'TIMESTAMP'
+  | 'GEOMETRY'
+  | 'POINT'
+  | 'LINESTRING'
+  | 'POLYGON'
+  | 'MULTIPOINT'
+  | 'MULTILINESTRING'
+  | 'MULTIPOLYGON'
+  | 'GEOMETRYZ'
+  | 'POINTZ'
+  | 'LINESTRINGZ'
+  | 'POLYGONZ'
+  | 'MULTIPOINTZ'
+  | 'MULTILINESTRINGZ'
+  | 'MULTIPOLYGONZ'
+  | 'GEOMETRYCOLLECTION';
```

Comment:
@saagar-cognite we are supporting more types (M and ZM geometries)
the list can be found in documentation: https://pr-1323.specs.preview.cogniteapp.com/v1.json.html#operation/createFeatureTypes